experimental/dds/tree2/src/simple-tree/rawNode.ts

```diff
@@ -0,0 +1,276 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { FieldKey, TreeNodeSchemaIdentifier } from "../core";
+import {
+	EditableTreeEvents,
+	FieldNodeSchema,
+	FlexTreeContext,
+	FlexTreeField,
+	FlexTreeNode,
+	FlexTreeTypedNode,
+	FlexTreeEntityKind,
+	FlexTreeMapNode,
+	FlexTreeObjectNode,
+	FlexTreeTypedField,
+	FlexTreeUnboxField,
+	FlexibleFieldContent,
+	boxedIterator,
+	flexTreeMarker,
+	onNextChange,
+	LocalNodeKey,
+	MapNodeSchema,
+	ObjectNodeSchema,
+	TreeNodeSchema,
+	TreeNodeSchemaBase,
+	TreeStatus,
+	FlexTreeFieldNode,
+} from "../feature-libraries";
+import { fail } from "../util";
+import { InsertableTypedNode } from "./insertable";
+
+/** Stores the content of the raw node, i.e. the data that was passed to the factory */
+const nodeContent = Symbol();
```

Comment:
Nit: This variable name feels a bit misleading to me. IMO `nodeContentSymbol` would make it a bit less confusing. But if this better matches the conventions we've been using, please disregard.

Reply:
I don't like the convention of putting `Symbol` on the end of symbols, it makes them too long and ugly. I don't think we have an established pattern - this is replacing a symbol that was also named `nodeContent`.